Tests/Foundation/TestURLSession.swift

```diff
@@ -652,6 +652,9 @@ final class TestURLSession: LoopbackServerTest, @unchecked Sendable {
     }
     
     func test_repeatedRequestsStress() async throws {
+        #if os(Windows)
+        throw XCTSkip("This test is currently disabled on Windows")
```

Comment:
This was a new test added after the re-core, so the test hadn't been run on Windows CI before. It's failing and @jrflat is looking into it